apps/prairielearn/src/components/AssessmentRegenerate.html.ts

```diff
@@ -27,13 +27,12 @@ export function RegenerateInstanceModal({ csrfToken }: { csrfToken: string }) {
 
 export function RegenerateInstanceAlert() {
   return html`
-    <div class="alert alert-warning alert-dismissible fade show">
-      You are viewing your own assessment instance and have instructor permissions.
+    <div class="alert alert-warning border-warning alert-dismissible fade show">
+      Instructors:
```

Comment:
In keeping with the "Staff information panel" I'm thinking this should actually say:
```suggestion
      Course staff:
```